### DISCUSSION TITLE
OKD UPI with internal load balancing apiVIPs and ingressVIPs

### DISCUSSION THREAD

**Discussion by borazem** [okd-project/okd#1859] [Q&A]
Dear all,

can you please help me understand why apiVIP and ingressVIP applies only to installer-provisioned infrastructure  as stated in https://docs.okd.io/latest/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere?

If I use that parameter in install-config.yaml and proceed with UPI deployment it works just fine, so would really like to understand why one could not use that in the UPI deployment.

Moreover, this functionality would be very useful also with anyplatform/baremetal deployment.

Thank you very much

**Comment by vrutkovs** [comment]
>f I use that parameter in install-config.yaml and proceed with UPI deployment it works just fine, so would really like to understand why one could not use that in the UPI deployment.

I think this is a documentation issue - this is _required_ when setting up IPI, but if set would be used on UPI installs too.

**Reply by borazem** [reply]
Thank you very much @vrutkovs.
